### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by amitkumarj441**
Signed-off-by: Amit Kumar Jaiswal <amitkumarj441@gmail.com>

### CODE REVIEW

File: testing/README.md [hunk 1]
```diff
@@ -27,7 +27,11 @@ CLUSTER=kubeflow-testing
 NAMESPACE=kubeflow-test-infra
 ```
 
-After starting `kubectl proxy` you can connect to it at
+After starting `kubectl proxy` you can connect to it at 
+
+```
+http://127.0.0.1:8001/api/v1/proxy/namespaces/kubeflow-test-infra/services/argo-ui/
```
Comment (foxish) [inline testing/README.md]: Remove the reference to the same URL below?

Comment (amitkumarj441) [reply testing/README.md]: Thanks @foxish for the feedback.

File: testing/README.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ CLUSTER=kubeflow-testing
 NAMESPACE=kubeflow-test-infra
 ```
 
-After starting `kubectl proxy` you can connect to it at
+After starting `kubectl proxy` you can connect to it at
```
Comment (foxish) [inline testing/README.md]: After starting `kubectl proxy` on `127.0.0.1:8001`, you can connect to the argo UI via the local proxy at: <url here>

Comment (amitkumarj441) [reply testing/README.md]: Updated accordingly. Please have a look at it.

**Comment by foxish** [comment]
Thanks!

**Comment by amitkumarj441** [comment]
Thanks a lot @foxish :smile:

**Pull request merged**